docs/infra/merge_queue.md

```diff
@@ -16,19 +21,36 @@ With the merge queue, the button will now look like this:
 
 
 Unlike "Squash and merge", the new button does not immediately land the PR on the target branch, but instead enqueues your PR for extra checks in the merge queue. If you do not do anything, eventually your PR will land.
-Can I still use autosubmit?
-Yes! Feel free to continue using the autosubmit label as normal. The bots will run your PR through the merge queue and land it automatically.
-What's "Remove from queue"? Can I click it?
+
+---
+
+### Can I still use autosubmit?
+
+Yes! Feel free to continue using the `autosubmit` label as normal. The bots will run your PR through the merge queue and land it automatically.
```

Comment:
Should we *encourage* people to use `autosubmit` instead of manually clicking the button?

Reply:
Folks already should be using the autosubmit label. Merging manually has been and is still discouraged.